### ISSUE TITLE
Rename to gnome-vertical-overview

### ISSUE THREAD

**Issue by romgrk** [RensAlthuis/vertical-overview#8]
Small change but it makes the project purpose clearer.

**Comment by RensAlthuis** [comment]
Hmm, this is a gnome extension though. I'll think about it, but I'm not sure if adding gnome- to the name really adds any new information.

**Comment by romgrk** [comment]
Ok ok, would be nice to update the description and tags though, will make the repo more clear (and hopefully attract more contributors).

![Screenshot from 2021-04-09 10-52-33](https://user-images.githubusercontent.com/1423607/114198808-bdaaf400-9921-11eb-9b9c-d146386b0b58.png)

**Comment by RensAlthuis** [comment]
Yeah, that's a good point. I'll add those, unless you have any specific suggestions?

**Comment by romgrk** [comment]
No, though is simple is always best I feel. While we're at it, it would also be nice to add a cool screenshot, or fix the video to make it work, here is how it renders on my side:

![image](https://user-images.githubusercontent.com/1423607/114202336-14fe9380-9925-11eb-9514-a0421767010c.png)


One more thing, probably the log-out/log-in in the instructions can be replaced by just restarting the shell with `Alt-F2`, then `r`.

**Comment by ismaail** [comment]
And don't forget to add it at https://extensions.gnome.org/

**Comment by AlexWayfer** [comment]
> Hmm, this is a gnome extension though. I'll think about it, but I'm not sure if adding gnome- to the name really adds any new information.

Honestly, all GNOME extensions on which I was working on somehow (or debug) have `gnome-` prefix. I don't know why exactly, but it makes easier sometimes to find / detect project. And it seems similar to the GNOME ecosystem, where are `gnome-shell`, `gnome-clocks`, `gnome-anything-else` (for example, KDE tries to add just `K`, like `KWin`, `KAlarm`, or even `Konsole`).

**Comment by RensAlthuis** [comment]
I was thinking of dash-to-dock, dash-to-panel, worspaces-to-dock, pixel-saver, arc-menu etc. 
But I am not really set on the name and this is not the hill I want to die on, so if I am alone in this I'll definitely change it.

**Comment by AlexWayfer** [comment]
> I was thinking of dash-to-dock, dash-to-panel, worspaces-to-dock, pixel-saver, arc-menu etc.
> But I am not really set on the name and this is not the hill I want to die on, so if I am alone in this I'll definitely change it.

These mentioned extensions are nice and popular. I also found emoji-selector among used by me. 

So… I don't think that this is important and anything important changing.

**Comment by romgrk** [comment]
> But I am not really set on the name and this is not the hill I want to die

Same here, if the choice was mine I'd go for `gnome-vertical-overview` for explicitness but I don't insist very much.
